mlflow/models/cli.py

```diff
@@ -155,6 +155,46 @@ def prepare_env(
     ).prepare_env(model_uri=model_uri)
 
 
+@commands.command("generate-dockerfile")
+@cli_args.MODEL_URI_BUILD_DOCKER
+@click.option(
+    "--output-directory",
+    "-d",
+    default="mlflow-dockerfile",
+    help="output directory for generating dockerfile",
+)
+@cli_args.ENV_MANAGER
+@cli_args.MLFLOW_HOME
+@cli_args.INSTALL_MLFLOW
+@cli_args.ENABLE_MLSERVER
+def generate_dockerfile(
+    model_uri, output_directory, env_manager, mlflow_home, install_mlflow, enable_mlserver
+):
+    """
+    Generates a directory with dockerfile whose default entrypoint serves an MLflow model at port
+    8080, using the python_function flavor. Function accepts a directory path and not just a
+    dockerfile path because if model is specified --- it will generate additional folder with
+    model files inside aforementioned directory This dockerfile defines an image that is
+    equivalent to the one produced by ``mlflow models build-docker``
+    """
```

Comment:
Can we add a small `logger.info("Generating Dockerfile)` message here? If a model_uri is specified, can we include it in the message?

Reply:
done